test/functional/connection/connection.ts

```diff
@@ -1,10 +1,16 @@
 import "reflect-metadata";
 import {expect} from "chai";
 import {Post} from "./entity/Post";
+import {Guest as GuestV1} from "./entity/v1/Guest";
+import {Comment as CommentV1} from "./entity/v1/Comment";
+import {Guest as GuestV2} from "./entity/v1/Guest";
+import {Comment as CommentV2} from "./entity/v1/Comment";
```

Comment:
In the previous test case, the result of loadTableSchemas is always empty. Because each time the establishment of connection will drop schema.
Only in this way can we cover the case of sync schema.